config/locales/import/csv/actiity_actual_refund_comment/errors.en.yml

```diff
@@ -0,0 +1,20 @@
+---
+en:
+  import:
+    csv:
+      activity_actual_refund_comment:
+        errors:
+          default:
+            required: Is required
```

Comment:
Is it correct for 'required' to be nested in 'default'?

Reply:
Yeah, it might read a bit strange! The idea is that `default` is not any specific value and `required` means the value will be required, there are a number of attributes that share this validation, we could repeat the `required` for each attribute but that felt like repetetion?

I am hoping someone with better content design will go over all these new error messages at some point - so I'll wait and see if the messages need to be different.